src/Polly.Shared/RetrySyntaxAsync.cs

```diff
@@ -19,6 +19,17 @@ public static partial class RetrySyntax
         /// <returns>The policy instance.</returns>
         public static Policy RetryAsync(this PolicyBuilder policyBuilder)
         {
+            return RetryAsync(policyBuilder, false);
+        }
+
+        /// <summary>
+        /// Builds a <see cref="Policy"/> that will retry once.
+        /// </summary>
+        /// <param name="policyBuilder">The policy builder.</param>
+        /// <param name="continueOnCapturedContext">Whether to continue on a captured synchronization context.</param>
+        /// <returns>The policy instance.</returns>
+        public static Policy RetryAsync(this PolicyBuilder policyBuilder, bool continueOnCapturedContext)
+        {
             return policyBuilder.RetryAsync(1);
```

Comment:
@yevhen This overload `RetryAsync(this PolicyBuilder policyBuilder, bool continueOnCapturedContext)` doesn't make use of the supplied `continueOnCapturedContext` value.  Does it want a small correction to:
`return policyBuilder.RetryAsync(1, continueOnCapturedContext);` ?

(Apologies @yevhen @joelhulen for intervening but hope useful - I spotted this while doing some exploratory investigation of how to add cancellation support!)